projects/plugins/videopress/changelog/update-jetpack-videopress-plugin-doc-improve

```diff
@@ -0,0 +1,4 @@
+Significance: minor
```

Comment:
Should this be a minor or a patch? :thinking:

Reply:
yes, I think it's a patch actually.